pkg/authorization/authorizer/messages.go

```diff
@@ -21,27 +21,28 @@ type ForbiddenMessageResolver struct {
 
 func NewForbiddenMessageResolver(projectRequestForbiddenTemplate string) *ForbiddenMessageResolver {
 	apiGroupIfNotEmpty := "{{if len .Attributes.GetAPIGroup }}{{.Attributes.GetAPIGroup}}.{{end}}"
```

Comment:
The dot is a separator for display.  It could be removed entirely and the messages reworked, but it can't move before.